Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gem 'hydra', '~> 7.1.0'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 4.1.4'
+gem 'blacklight', github: 'projectblacklight/blacklight', branch: 'release-5.5'
```

Comment:
@skng Why did we have to peg to blacklight 5.5 here? Were we using a different version before?